app/[locale]/institute/administration/(committees)/committee.tsx

```diff
@@ -120,6 +132,31 @@ export default async function Committee({
           ))}
         </TableBody>
       </Table>
+      <Pagination>
+        <PaginationContent>
+          <PaginationItem>
+            <PaginationPrevious
+              className={
+                Number(page) <= 1 ? 'pointer-events-none opacity-50' : undefined
```

Comment:
Is this for showing the arrow as disabled? If yes then there are 2 things wrong with this:
- You don't use `pointer-events-none` for this. Use `disabled:cursor-not-allowed` or `data-[disabled]:cursor-not-allowed` depending on the context.
- You won't define this logic here, rather within the pagination component itself.